From c1c15689705554805158b0a5437aed8a1541d5ec Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Carlos=20Ag=C3=BCero?= <cen.aguero@gmail.com>
Date: Thu, 20 Oct 2016 17:06:46 -0700
Subject: [PATCH 3/5] Tweak install directory

---
 CMakeLists.txt | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index fce2f3b..5d8e191 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,13 +8,7 @@ catkin_package(
 #  CATKIN_DEPENDS std_msgs
 #  DEPENDS system_lib
 )
-install(DIRECTORY .
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
-  FILES_MATCHING PATTERN "*.gradle"
-                 PATTERN "gradlew"
-  PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ
-                    GROUP_EXECUTE GROUP_READ
-)
+
 install(DIRECTORY gradle
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
 )
-- 
2.7.4

